packages/styles/src/components/form-validation.scss

```diff
@@ -1,116 +0,0 @@
-@forward './../variables/options';
-
-@use 'sass:map';
-
-@use './../themes/bootstrap/core' as b;
-@use './../themes/bootstrap/forms/validation' as bfv;
-
-@use './../mixins/color' as color-mx;
-@use './../mixins/utilities' as utilities-mx;
-@use './../mixins/form-validation' as form-validation-mx;
-@use './../variables/color';
-@use './../variables/components/form-validation';
-@use './../variables/components/forms';
-@use './../variables/spacing';
-@use './../variables/type';
-@use './../mixins/icons';
-
-.invalid-feedback,
-.valid-feedback {
-  display: none;
-  align-items: center;
-  width: 100%;
-  margin-block: form-validation.$form-feedback-margin-top 0;
```

Comment:
You've deleted a lot of variables from the _form-validation.scss variables file, make sure to delete the ones that are no longer used anywhere. You can probably even delete the file if none are used anymore